PlatformParticles/PlatformParticles/_Actions/DismissPresentedAction.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DismissPresentedAction.swift
+//  PlatformParticles
+//
+//  Created by Mike Maguire on 9/15/24.
+//  Copyright © 2024 dYdX. All rights reserved.
+//
+
+import ParticlesKit
+import RoutingKit
+import Utilities
+
+public class DismissPresentedActionBuilder: NSObject, ObjectBuilderProtocol {
+    public func build<T>() -> T? {
+        let action = DismissPresentedAction()
+        return action as? T
+    }
+}
+
+private class DismissPresentedAction: NSObject, NavigableProtocol {
```

Comment:
needed a new dismiss action which dismisses instead of first trying to pop like `/action/dismiss`

Reply:
Maybe better to reuse /action/dismiss and add an optional param of RoutingRequest to specify this behavior.  This way, we don't have to maintain two different "dismiss" actions.